### PR TITLE
Move news to new database

### PR DESCRIPTION
**Pull request by ThiefMaster**

### CODE REVIEW

File: indico/MaKaC/webinterface/tpls/CategoryDisplay.tpl [hunk 1]
```diff
@@ -93,12 +93,16 @@ containsCategories = len(categ.getSubCategoryList()) > 0
...
-            % if isNewsActive:
+            <%
+            from indico.modules.news.util import get_recent_news
+            news = get_recent_news()
+            %>
+            % if news:
                 <h2 class="icon-bullhorn">
                     ${_("News")}
-                    <a href="${ urlHandlers.UHIndicoNews.getURL()}" class="more-icon">${ _("more...") }</a>
+                    <a href="${url_for('news.display')}" class="more-icon">${ _("show all...") }</a>
```
Comment (nop33) [inline indico/MaKaC/webinterface/tpls/CategoryDisplay.tpl]: Here you didn't leave a space before `url_for`, but later on in `indico/MaKaC/webinterface/tpls/WelcomeHeader.tpl` line 6 you do.

Comment (ThiefMaster) [inline indico/MaKaC/webinterface/tpls/CategoryDisplay.tpl]: Legacy template, goes away before the release. But yeah I'll change it

File: indico/modules/news/controllers.py [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+from indico.modules.news import news_settings, logger
+from indico.modules.news.forms import NewsSettingsForm, NewsForm
+from indico.modules.news.models.news import NewsItem
+from indico.modules.news.views import WPNews, WPManageNews
+from indico.util.date_time import now_utc
+from indico.util.i18n import _
+from indico.web.flask.util import url_for
+from indico.web.forms.base import FormDefaults
+from MaKaC.webinterface.rh.base import RH
+from indico.web.util import jsonify_data, jsonify_form
```
Comment (nop33) [inline indico/modules/news/controllers.py]: Wrong order

File: indico/modules/news/views.py [hunk 3]
```diff
@@ -14,20 +14,26 @@
...
-
-    def _process( self ):
-        p = news.WPNews(self)
-        return p.display()
-
+from __future__ import unicode_literals
 
+from MaKaC.webinterface.pages.admins import WPAdminsBase
+from indico.util.i18n import _
+from MaKaC.webinterface.pages.base import WPJinjaMixin, WPDecorated
```
Comment (nop33) [inline indico/modules/news/views.py]: Wrong order too

**Pull request merged**